docs/reference/release-notes/6.8.asciidoc

```diff
@@ -47,9 +47,12 @@ Features/Watcher::
 * Fix class used to initialize logger in Watcher {pull}46467[#46467]
 
 Machine Learning::
-* [ML] Reinstate ML daily maintenance actions {pull}47103[#47103] (issue: {issue}47003[#47003])
-* [ML] fix two datafeed flush lockup bugs {pull}46982[#46982]
-* [ML] Throw an error when a datafeed needs CCS but it is not enabled for the node {pull}46044[#46044] (issue: {issue}46025[#46025])
+* Reinstate ML daily maintenance actions {pull}47103[#47103] (issue: {issue}47003[#47003])
+* Fix two datafeed flush lockup bugs {pull}46982[#46982]
+* Throw an error when a datafeed needs CCS but it is not enabled for the node {pull}46044[#46044]
+* A reference to a temporary variable was causing forecast model restoration to 
+fail. The bug exhibited itself on MacOS builds with versions of clangd > 10.0.0
+{ml-pull}688[#688]
```

Comment:
From an end user perspective this one should have been something more like:

* Fix possibility of crash when calculating forecasts that overflow to disk. {ml-pull}688[\#688]

The thing about clang 10+ refers to how it was reproducible in the unit tests.  The fix was for undefined behaviour so the bug could affect the production code differently.